buildpacks/nodejs-function-invoker/src/function.rs

```diff
@@ -0,0 +1,132 @@
+use heroku_nodejs_utils::package_json::{PackageJson, PackageJsonError};
+use libcnb::read_toml_file;
+use libherokubuildpack::toml_select_value;
+use std::path::PathBuf;
+use thiserror::Error;
+use toml::Value;
+
+pub fn is_function<P>(d: P) -> bool
+where
+    P: Into<PathBuf>,
+{
+    let dir: PathBuf = d.into();
```

Comment:
Ultra-nit: the explicit type can be removed since `Into<PathBuf>` is the only `Into` available for `P`.